sharan16/HelpMeLabel
# Created a post request for labeling images

When you hit this url  you will be able to label an image.

How to test this?
Use postman, set request to Post, set address to http://localhost:8000/api/images/label_image and send details in body like shown below in the image. Once the request has been sent check it out in the db ;P

[ch20]

![image](https://user-images.githubusercontent.com/12437329/79707213-afd00f00-8289-11ea-9ffd-4c06274c8ab6.png)
